conftest.py

```diff
@@ -23,6 +23,8 @@
 test_launcher = CONFIG.test_launcher
 test_device = CONFIG.test_device
 test_nic = CONFIG.test_interface
+test_port = CONFIG.test_port
```

Comment:
Can we add a line about the used port in `print_test_configuration`? And maybe mention that ports `test_port` and `test_port+1` will be used (see another comment in this review)?